plugins/bufbuild/validate-cpp/v0.9.0/buf.plugin.yaml

```diff
@@ -3,6 +3,8 @@ name: buf.build/bufbuild/validate-cpp
 plugin_version: v0.9.0
 source_url: https://github.com/bufbuild/protoc-gen-validate
 description: Generates C++ code to validate Protobuf messages using protoc-gen-validate constraints.
+deps:
+  - plugin: buf.build/protocolbuffers/cpp:v21.12
```

Comment:
I started by pinning the latest available version. We can update when new versions of these plugins are updated (but I don't think it is worth updating each time a dependency updates).